draft-scenarios/clash/gold_rush.tex

```diff
@@ -27,113 +31,73 @@ \subsection*{\MakeUppercase{Player Setup}}
   \item A Few \silver\ Units with the lowest Recruitment cost
 \end{itemize}
 
-\textbf{Town Buildings:} \bronze\ Dwelling
+\textbf{Town Buildings:} \bronze\ Dwelling, Citadel
+
+\textbf{Map Tile Pool:} Each player takes 2 Far (II--III) Map Tiles.
 
-\textbf{Map Tile Pool:} Each player takes 3 Far (II--III) Map Tiles with at least one Settlement.
+% It looks much better if Map Setup section is in new column.
+% Both \vfill or \columnbreak work strangely here.
+% Maybe replace it with some fancy image in the future.
+\vspace{9em}
```

Comment:
`\vspace*{\fill}` gets the job done. But instead, please put some art here. The elemental conflux from the new rule books will fit in great.
![elemental_conflux](https://github.com/user-attachments/assets/d9d9f6db-fbe3-40ea-91a6-af2bac6cd6c9)

Reply:
See example: https://github.com/qwrtln/Homm3BG-mission-book/blob/7e48327ce127e594ceaef20fb76c5a29c690b62f/campaigns/inferno_devilish_plan.tex#L84